ipa/scripts/cli.py

```diff
@@ -234,6 +234,10 @@ def main(context, no_color):
     '--vnet-resource-group',
     help='Azure resource group name where virtual network is found.'
 )
+@click.option(
```

Comment:
This should be a flag. See `--no-color` example.

```
@click.option(
    '--no-color',
    is_flag=True,
    help='Remove ANSI color and styling from output.'
)
```

Reply:
done